Tools/TestWebKitAPI/Tests/WTF/StringCommon.cpp

```diff
@@ -196,7 +239,11 @@ TEST(WTF_StringCommon, FindIgnoringASCIICase)
     EXPECT_EQ(WTF::findIgnoringASCIICase(unsafeSpanChar8("🍉WaterMelon🍉"), "water"_s), 4UZ);
     EXPECT_EQ(WTF::findIgnoringASCIICase(unsafeSpanChar8("Water🍉Melon"), unsafeSpanChar8("🍉")), 5UZ);
     EXPECT_EQ(WTF::findIgnoringASCIICase(unsafeSpanChar8("🍉WaterMelon🍉"), unsafeSpanChar8("🍉")), 0UZ);
-    // EXPECT_NEQ(WTF::findIgnoringASCIICase("test"_spanChar8, "test"_span8), notFound); // This should not compile.
+    EXPECT_EQ(WTF::findIgnoringASCIICase(std::span<const char8_t>(), std::span<const char8_t>()), 0UZ);
+    // EXPECT_EQ(WTF::findIgnoringASCIICase(std::span<const char8_t>(), unsafeSpanChar8("🍉WaterMelon🍉")), notFound);
+    // EXPECT_EQ(WTF::findIgnoringASCIICase(""_spanChar8, unsafeSpanChar8("🍉WaterMelon🍉")), notFound);
+    // EXPECT_EQ(WTF::findIgnoringASCIICase(unsafeSpanChar8("🍉"), unsafeSpanChar8("🍉WaterMelon🍉")), notFound);
```

Comment:
Were these lines left commented accidentally? If they are supposed to not compile, please add a comment mentioning that.